package/yast2-network.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Aug 18 08:52:41 UTC 2016 - mfilka@suse.com
+
+- bnc#991486
+  - extended fix for this bug from 3.1.163. Do not report invalid
+    names incorrectly
```

Comment:
"Do not validate existing interfaces hardware" (or the converse) is too generic, I would not understand what is being fixed.
Another try:

>   Do not report "Configuration name is invalid" in Network Card Setup / Hardware when editing an existing configuration